pkg/apis/resource/types.go

```diff
@@ -322,6 +324,66 @@ type Device struct {
 	// +listType=atomic
 	// +featureGate=DRADeviceTaints
 	Taints []DeviceTaint
+
+	// UsageRestrictedToNode indicates if the usage of an allocation involving this device
+	// has to be limited to exactly the node that was chosen when allocating the claim.
+	//
+	// This is an alpha field and requires enabling the DRADeviceBindingConditions and DRAResourceClaimDeviceStatus
+	// feature gates.
+	//
+	// +optional
+	// +featureGate=DRADeviceBindingConditions,DRAResourceClaimDeviceStatus
+	UsageRestrictedToNode *bool
```

Comment:
`BindsToNode` sounds best to me (from https://github.com/kubernetes/kubernetes/pull/130160/files#r2201675405).

It ties the field to the corresponding feature and the present tense isn't wrong like "bound" was.